docs/FAQ/gui-faq-en.md

```diff
@@ -20,147 +20,155 @@ Zonemaster
+Yes. If DNSSEC is available for a domain that is tested by Zonemaster, it will be
 checked automatically.
+Note that you can also explicitly provide Delegation Signers (DS) records using the
+"Options" button.
 
 <a name="q8"></a>
 #### 8. What makes Zonemaster differ from other DNS zone validating software?
 Firstly, Zonemaster saves all history from earlier tests based on the tested
-domain, which means you can go back to a test you did a week ago and compare it
-to the test you ran a moment ago.
+domain, which means you can go back to a test you did some time ago and compare it
+to the test you ran just a moment ago.
 
-All tests that Zonemaster runs are defined in test case specifications that
-are linked from [Test Requirements
-document](https://github.com/zonemaster/zonemaster/blob/master/docs/requirements/TestRequirements.md)
+Secondly, all tests that Zonemaster runs are defined in Test Case specifications that
+can be found in the [Test Requirements] document.
 
-Zonemaster could be used to test undelegated domains. More about undelegated
-domains in [Question 12](#q12).
+Thirdly, Zonemaster can be used to test undelegated domains. Find more about undelegated
+domains in [Question 12].
 
 Lastly, this open source version of Zonemaster was built using modular code
-which, basically, means you can use parts of it in your systems, if you would want
-to. It is quite rare that you'd want a complete program just to check for example
+which basically means that you can integrate parts of it in your own systems, if you wish.
+For example, it is quite rare that you would want a complete program just to check for
 redelegations.
 
 <a name="q9"></a>
 #### 9. Zonemaster and privacy
 Since Zonemaster is open to everyone it is possible for anyone to check your
-domain and also see previous tests, however there is no way to tell
-who has run a specific test since nothing is logged except the time of the test.
+domain and its tests history.
+However there is no way to tell who has run a specific test since nothing is
+logged except the start time of the test.
 
 <a name="q10"></a>
-#### 10. How come I cannot test my domain?
-If we disregard the situation where the domain does not exist, as in when you input a
-non-existing domain to Zonemaster, there are 2 other possibilites:
-  - To protect the engine from multiple identical inputs, that is the same IP
-    checking the same zone several times, there is a delay of 5 minutes between
-    identical subsequent tests. This means that you can only test the
-    same domain once every 5 minutes, if you try and test it again within 5 minutes
-    the last results will be displayed instead.
-  - Because Zonemaster was made to check domains or DNS zones, like zonemaster.net, and
-    not hostnames in a domain (zone), like www.zonemaster.net, the Zonemaster will
-    report a failure if you try to test a host name instead of a domain matching a
+#### 10. How come my domain cannot be tested?
```

Comment:
1. Your domain name is not yet delegated.
2. Your domain name is not reachable from public Internet.
3. Zonemaster can only test what is called DNS zone (e.g. zonemaster.net) and not host names (e.g. www.zonemaster.net)
4. You have misspelled your domain name.

Reply:
Updated.